test/ResultTableColumnOperationsTest.cpp

```diff
@@ -290,7 +339,12 @@ TEST(ResultTableColumnOperations, SumUpColumns) {
                                        createDummyValueEntryType<T>()),
           table, ColumnNumWithType<T>{9});
     }
-  });
+  }
+};
+
```

Comment:
This test file also looks like the diff gets much smaller if we rewrite the `doForTypeInResult` to make it use a `type_identity`.

Reply:
Addressed